Materials/Types/MinimalMultilayerPBR_LayerProperties.json

```diff
@@ -0,0 +1,75 @@
+{
+    "description": "Settings that affect one layer of the multilayer material.",
+    "properties": [
+        {
+            "name": "metallic",
+            "displayName": "Metallic",
+            "description": "Controls how metallic the surface appears.",
+            "type": "Float",
+            "defaultValue": 0.0,
+            "min": 0.0,
+            "max": 1.0,
+            "connection": {
+                "type": "ShaderInput",
+                "name": "metallic"
+            }
+        },
+        {
+            "name": "roughness",
+            "displayName": "Roughness",
+            "type": "Float",
+            "defaultValue": 1.0,
+            "min": 0.0,
+            "max": 1.0,
+            "connection": {
+                "type": "ShaderInput",
+                "name": "roughness"
+            }
+        }
+    ],
+    "propertyGroups": [
+        {
+            "name": "baseColor",
```

Comment:
Can we nest imports, so that this imports the baseColor properties from another file (that is in turn, also used by StandardPBR)? If so, it might be worth doing that here to validate that importing a file which in turn imports another file is something that works.

Reply:
Yes we can nest them. I have a unit tests to verify this. I'll exercise the nesting in actual data when I refactor our main material types. My goal for MinimalMultilayerPbr was more around testing multi-level nesting and name contexts in Material Editor and Material Component UI and .material file saving.